java/ql/src/semmle/code/java/security/XSS.qll

```diff
@@ -77,7 +80,7 @@ class XssSink extends DataFlow::ExprNode {
   }
 }
 
-class ServletWriterSourceToWritingMethodFlowConfig extends TaintTracking::Configuration {
+private class ServletWriterSourceToWritingMethodFlowConfig extends TaintTracking2::Configuration {
```

Comment:
What's the reason for using `TaintTracking2` versus `TaintTracking` ?

Reply:
The result of this configuration is used in the definition of `XssSink`. So this config must come from a different library copy than any config that uses `XssSink` (otherwise we'd have negative recursion through the dataflow library).  In this PR I simply switch use of the two copies around, such that the queries can use the default copy.